docs/spec/WIP_provisioning/overview.md

```diff
@@ -0,0 +1,229 @@
+# Fee Distribution
+
+## Overview
+
+Fees are pooled separately and withdrawn lazily, at any time.  They are not
```

Comment:
Separately from what? Accounts?

Consider rephrasing, e.g. "Received transaction fees are stored temporarily in a pool. Fees can be withdrawn from this pool at will by delegators & validators in proportion to their fee allocations."

Reply:
WIP, next PR